src/Strategy/RPM.hs

```diff
@@ -77,11 +76,11 @@ findProjects = walkWithFilters' $ \dir _ files -> do
 
   case specs of
     [] -> pure ([], WalkContinue)
-    _ -> pure ([RpmProject dir specs], WalkContinue)
+    _ -> pure (RpmProject dir <$> specs, WalkContinue)
 
 data RpmProject = RpmProject
   { rpmDir :: Path Abs Dir
-  , rpmFiles :: [Path Abs File]
+  , rpmFiles :: Path Abs File
```

Comment:
Nit: should we rename this to `rpmFile`, since it's just one file now?

Reply:
Yes, good catch